tt_metal/include/compute_kernel_api/bcast.h

```diff
@@ -126,7 +126,7 @@ ALWI void add_tiles_bcast_scalar(uint32_t icb0, uint32_t icb1, uint32_t itile0,
  * | ocb            | The indentifier of the circular buffer (CB) containing output | uint32_t      | 0 to 31 | False |
  */
 template <EltwiseBinaryType tBcastOp, BroadcastType tBcastDim>
-void init_bcast(uint32_t icb0, uint32_t icb1, uint32_t ocb = 16) {
+void init_bcast(uint32_t icb0, uint32_t icb1, uint32_t ocb) {
     if constexpr (tBcastOp == ELWMUL) {
```

Comment:
The issue for which this PR is made mentions in the title that default value of output buffer should be removed but in the description it mentions default value for the circular buffers should be removed altogether (which makes sense) because default values of all circular buffers are equally likely to cause issues. Now on one hand a PR should address one issue only. But if the goal of this PR is to fix one file at a time, this file has many more default values left. Do you want to handle them in separate PR's ? I am approving this PR assuming that you plan that way.

Reply:
My initial understanding was that only the output operands needed cleaning up. I'll address the rest of the operands in a separate PR. Thanks for pointing it out